tools/integration_tests/read_cache/setup_test.go

```diff
@@ -99,8 +99,12 @@ func mountGCSFuseAndSetupTestDir(flags []string, ctx context.Context, storageCli
 	testDirPath = client.SetupTestDirectory(ctx, storageClient, testDirName)
 }
 
-func createConfigFile(cacheSize int64, cacheFileForRangeRead bool, fileName string, enableParallelDownloads bool) string {
-	cacheDirPath = path.Join(setup.TestDir(), cacheDirName)
+func getDefaultCacheDirPathForTests() string {
+	return path.Join(setup.TestDir(), cacheDirName)
+}
+
+func createConfigFile(cacheSize int64, cacheFileForRangeRead bool, fileName string, enableParallelDownloads bool, customCacheDirPath string) string {
```

Comment:
nit: can we directly pass parameter as cacheDirPath and use it (instead of using local cacheDirPath variable)?

Reply:
`cacheDirPath` is not a local variable to this function and is used at other places also. that's why I had to use like this. 

The other option is to set cacheDirPath outside of this function and then pass it the way you are mentioning but that would require changes in all test files.